sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -382,7 +382,7 @@ abstract class OffsetWindowFunction
  *
  * @param input expression to evaluate 'offset' rows after the current row.
  * @param offset rows to jump ahead in the partition.
- * @param default to use when the input value is null or when the offset is larger than the window.
+ * @param default to use when the offset is larger than the window.
```

Comment:
@hvanhovell what was the reason that we changed the behavior of lead and lag on if they respect null values?